cdc/model/mounter.go

```diff
@@ -71,3 +83,19 @@ func ComparePolymorphicEvents(i, j *PolymorphicEvent) bool {
 	}
 	return i.CRTs < j.CRTs
 }
+
+// ResolvedTs is the resolved timestamp of sink module.
+type ResolvedTs struct {
+	Ts    uint64
+	Batch bool
+}
+
+// NewResolvedTs creates a new ResolvedTs.
+func NewResolvedTs(t uint64) ResolvedTs {
+	return ResolvedTs{Ts: t, Batch: false}
+}
+
+// NewBatchResolvedTs creates a ResolvedTs with a given batch type.
+func NewBatchResolvedTs(t uint64, b bool) ResolvedTs {
```

Comment:
It looks that `NewBatchResolvedTs` can directly return a `ResolvedTs` with `Batch` is `True`.